### PR TITLE
applications: nrf_desktop: Configure PMW3360 via DTS

### PR DESCRIPTION
**Pull request by pdunaj**
Jira:DESK-598

### CODE REVIEW

**Comment by pdunaj** [comment]
Tested on PCA20041 - everything works as usual.

**Comment by pdunaj** [comment]
Once this is done I will do similar change for PAW3212.

**Comment by pdunaj** [comment]
Hi @carlescufi , @anangl ,  I have assigned @anangl as codeowner for dts folder as this matches closely with drivers (and I had to set something for CI to finish).
Please tell if I should change this.

**Comment by MarekPieta** [comment]
Remove PMW3360 HW config commit message - shouldn't it be "it is provided"?

File: drivers/sensor/pmw3360/pmw3360.c [hunk 1]
```diff
@@ -135,8 +144,8 @@ struct pmw3360_data {
 static const struct spi_config spi_cfg = {
 	.operation = SPI_WORD_SET(8) | SPI_TRANSFER_MSB |
 		     SPI_MODE_CPOL | SPI_MODE_CPHA,
-	.frequency = CONFIG_DESKTOP_SPI_FREQ_HZ,
-	.slave = 0,
+	.frequency = DT_INST_0_PIXART_PMW3360_SPI_MAX_FREQUENCY,
```
Comment (MarekPieta) [inline drivers/sensor/pmw3360/pmw3360.c]: consider using define's here too

Comment (pdunaj) [reply drivers/sensor/pmw3360/pmw3360.c]: I prefer to leave it. I used macros in other place to make code lines shorter. Here it does not mater much really.

**Comment by pdunaj** [comment]
I have fixed commit message after @MarekPieta comment. Also increased the SPI frequency as pointed out by @anangl offline.

**Comment by carlescufi** [comment]
> Hi @carlescufi , @anangl , I have assigned @anangl as codeowner for dts folder as this matches closely with drivers (and I had to set something for CI to finish).
> Please tell if I should change this.

That is perfectly fine, thank you for checking

**Comment by pdunaj** [comment]
Hi @wiminordic , do you know why CI is taking so long?

**Comment by pdunaj** [comment]
Hi @carlescufi , can this be merged?

**Comment by carlescufi** [comment]
@pdunaj sorry for the late reply, yes, it can be merged as I see you did

**Pull request merged**